.projections.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
what is this?

Reply:
https://marketplace.visualstudio.com/items?itemName=will-wow.vscode-alternate-file
or https://github.com/tpope/vim-projectionist
I have a shortcut for moving between implementation and test files and thought it might be useful for others too.
(My shortcut is <space>gt  (go to test))